compiler/rustc_mir/src/dataflow/framework/lattice.rs

```diff
@@ -104,7 +104,7 @@ impl MeetSemiLattice for bool {
 /// of the least upper bounds of each element of the tuple (or list).
 ///
 /// In other words:
-///     (A₀, A₁, ..., Aₙ) ∨ (B₀, B₁, ..., Bₙ) = (A₀∨B₀, A₁∨B₁, ..., Aₙ∨Bₙ)
+///     (A₀, A₁, ..., An) ∨ (B₀, B₁, ..., Bn) = (A₀ ∨ B₀, A₁ ∨ B₁, ..., An ∨ Bn)
```

Comment:
I changed this because the character that was there before (a subscript 'n') doesn't seem to be commonly available in fonts. It doesn't render in the compiler docs, GitHub, or my terminal.

![image](https://user-images.githubusercontent.com/37223377/98064543-69966380-1e07-11eb-8a0d-4f1db4f3e7cc.png)

Reply:
You can also use `<sub></sub>` tags.

Reply:
Doesn't seem to render on edge either
![image](https://user-images.githubusercontent.com/4193035/98101081-bd349b80-1e5f-11eb-82e5-3a5e43abfeaa.png)

Reply:
in my browser it says it is using Noto Sans Mono, and it renders correctly

Reply:
Are the docs somehow forcing a bad font?  Shows up fine here in FF:
![image](https://user-images.githubusercontent.com/18526288/98482091-cb374300-21f6-11eb-993f-9adaea2aa08a.png)

Reply:
It looks fine in Safari on my phone, but doesn't render correctly in Firefox on my computer. I think some fonts do have this character but enough don't such that we shouldn't use the character.

Reply:
Marking this as resolved so we don't bikeshed too much ;)